RELEASE.md

```diff
@@ -1,3 +1,40 @@
+# Release 0.1.0
+
+## New features
+- Telemetry. Marqo now includes various timing metrics for many API endpoints when 
```

Comment:
Let's make this more precise:
"... many API endpoints..." -> "... for the `search`, `bulk search` and `add_documents` endpoints ..."